roles/container_runtime/defaults/main.yml

```diff
@@ -22,8 +22,11 @@ openshift_docker_insecure_registries: []
 openshift_docker_ent_reg: 'registry.access.redhat.com'
 
 openshift_docker_options: False  # str
-openshift_docker_log_driver: False  # str
-openshift_docker_log_options: []
+openshift_docker_log_driver: "journald"  # str
```

Comment:
I'm not sure if this is the default or not in RHEL.  Please confirm.

Reply:
@cevich checked pn a RHEL 7.5 machine and  /etc/sysconfig/docker has --log-driver=journald by default. But, setting the log-driver to journald is causing the prow and jenkins/install tests to fail. So not sure what is happening there.
The tests pass now though, with no log-driver being set and only setting the max-size when the driver is json.